app/scenes/Document/components/EditableTitle.tsx

```diff
@@ -15,9 +15,15 @@ import { isModKey } from "~/utils/keyboard";
 type Props = {
   value: string;
   document: Document;
+  /** Should the title be editable, policies will also be considered separately */
```

Comment:
These comments show up when you use the component, so that's pretty nice. We should make more use of this going forward!